contracts/ModuleRegistry.sol

```diff
@@ -149,6 +147,9 @@ contract ModuleRegistry is IModuleRegistry, EternalStorage {
      * @param _moduleFactory is the address of the module factory to be registered
      */
     function registerModule(address _moduleFactory) external whenNotPausedOrOwner {
+        if (IFeatureRegistry(getAddress(Encoder.getKey('featureRegistry'))).getFeatureStatus("customModulesAllowed")) {
+            require(msg.sender == getAddress(Encoder.getKey("owner")), "Only owner allowed to register modules");
```

Comment:
@adamdossa this should also say `|| IOwnable(_moduleFactory).owner() == IOwnable(msg.sender).owner()`
so even when the "customModulesAllowed" is set, Polymath can still register someone else's module on their behalf, if for some reason we wanted so.

Reply:
Actually, the require is wrong, it should be:

```
require(msg.sender == IOwnable(_moduleFactory).owner() || msg.sender == getAddress(Encoder.getKey('owner')),
        "msg.sender must be the Module Factory owner or registry curator");
```